packages/angular-material/tsconfig.spec.json

```diff
@@ -11,11 +11,5 @@
   "include": [
     "**/*.spec.ts",
     "**/*.d.ts"
-  ],
-  "angularCompilerOptions": {
-    "compilationMode": "partial",
-    "strictMetadataEmit": true,
-    "generateCodeForLibraries": false,
-    "skipTemplateCodegen": true
```

Comment:
Why did you remove this? Are these all defaults?

Reply:
I must admit I don't think they are necessary anymore. If I'm not mistaken, the only one still present when generating a lib with the angular cli now is `generateCodeForLibraries` for the production build.

In doubt, I'll add them again.